pyuvdata/uvbeam/tests/test_uvbeam.py

```diff
@@ -20,17 +20,14 @@
 import pyuvdata.utils as uvutils
```

Comment:
There's another `from __future__` import at the top of this file that should be removed.